xbmc/addons/addoninfo/AddonInfo.h

```diff
@@ -134,7 +134,7 @@ struct DependencyInfo
   }
 };
 
-typedef std::map<std::string, std::string> InfoMap;
+using InfoMap = std::map<std::string, std::string, std::less<>>;
```

Comment:
This was existing code, but is it a problem that `InfoMap` is defined in two headers?

Reply:
It’s not a problem as long as both definitions are identical. Otherwise compiler will bail out.